pkg/api/client.go

```diff
@@ -23,7 +23,7 @@ const (
 	PUT    string = http.MethodPut
 	DELETE string = http.MethodDelete
 
-	defaultTimeout     = time.Second * 60
+	defaultTimeout     = time.Second * 30
```

Comment:
Was this meant to change?

Reply:
Yes; the story asks for 30 second defaults. I contemplated adding a new variable.

Reply:
This is good then.

Reply:
sorry, but did C know that it was at 60 seconds?  I know the story said 30, but I think we moved it to 60 to accommodate the huge petstore swagger suman was testing (if i remember correctlY)